source/style.css

```diff
@@ -0,0 +1,1710 @@
+
+  background: #6070ff;
+}
+
+.my-contact {
+  /* Primary Text */
+
+  display: flex;
+  height: 150px;
+  padding-top: 2em;
+
+  /* Header / 2 */
+
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+
+  /* or 130% */
+
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+
+  /* Neutrals/N0 */
+
+  color: #fff;
+}
+
+.my_info {
+  /* Primary Text */
+
+  position: static;
+  width: 569px;
+  height: 102px;
+  padding-left: 10em;
+  top: 50px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+
+  /* or 140% */
+
+  text-align: center;
+
+  /* Primary/B50 */
+
+  color: #ebebff;
+
+  /* Inside Auto Layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 1;
+  margin: 12px 0.02px;
+}
+
+input[type="text"] {
+  position: absolute;
+  margin-left: 17em;
+  margin-top: 5em;
+  background: #fff;
+
+  /* Base/100 */
+
+  border: 1px solid #cfd8dc;
+  border-radius: 2px;
+}
+
+#name {
+  position: absolute;
+  display: flex;
+  width: 23em;
+  left: 16px;
+  right: 20px;
+  height: 40px;
+  top: 25em;
+  text-align: left;
+  align-items: center;
+
+  /* Mono/White */
+
+  background: #fff;
+
+  /* Base/100 */
+
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+input[type="email"] {
+  position: absolute;
+  margin-left: 17em;
+  margin-top: 5em;
+  background: #fff;
+
+  /* Base/100 */
+
+  border: 1px solid #cfd8dc;
+  border-radius: 2px;
+}
+
+#email {
+  position: absolute;
+  display: flex;
+  width: 23em;
+  left: 16px;
+  right: 20px;
+  height: 40px;
+  top: 30em;
+  text-align: left;
+  align-items: center;
+
+  /* Mono/White */
+
+  background: #fff;
+
+  /* Base/100 */
+
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+textarea[name="message"] {
+  /* Container */
+
+  position: absolute;
+  margin: 8px 0;
+  width: 25em;
+  height: 160px;
+  top: 41em;
+  margin-left: 18em;
+
+  /* Mono/White */
+
+  background: #fff;
+
+  /* Base/100 */
+
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+#message-form {
+  position: absolute;
+  left: 16px;
+
+  /* Input /Normal */
+
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+
+  /* or 160% */
+
+  /* Neutrals/N60 */
+
+  color: #b3bac5;
+}
+
+input[type="submit"] {
+  /* Auto Layout */
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 16px;
+  position: absolute;
+  width: 140px;
+  height: 48px;
+  margin-left: 25em;
+  top: 50.5em;
+  list-style: none;
+  color: #6070ff;
+  text-align: center;
+  justify-content: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+  border-radius: 8px;
+}
+
+input[type="submit"]:hover {
```

Comment:
## 🔴 

- [ ] Please refer to Classes and Id's instead of elements in your CSS file.